packages/nimble-components/src/table-column/text/styles.ts

```diff
@@ -5,7 +5,7 @@ import {
     controlLabelFontColor
 } from '../../theme-provider/design-tokens';
 
-export const cellStyles = css`
+export const cellViewStyles = css`
```

Comment:
Are we happy with the file structure within `table-column/text`? In general within nimble-components, `index.ts`, `template.ts`, and `styles.ts` all combine to define a component. In this case, `index.ts` is the column definition, and `cell-view.ts`, `template.ts`, and `styles.ts` all combine to define the cell view component. I personally find this a bit confusing/inconsistent. A few alternative suggestions could be:
1. Rename `styles.ts` to `cell-view-styles.ts` and rename `template.ts` to `cell-view-template.ts`
2. Make a subfolder under `table-column/text` named `cell-view` and move `cell-view.ts` there with the name `index.ts` and move `styles.ts` and `template.ts` there.
3. Rename `index.ts` to something else. One suggestion could be `table-column.ts`. Then, you could rename `cell-view.ts` to `index.ts`.

Reply:
I also find this more confusing then necessary. I'll add that in my ['Table row grouping' PR](https://github.com/ni/nimble/pull/1139), I created a sub-directory under '/table-column/text' called 'group-header' to host all of the standard files for that view.

Reply:
So following pattern 2 + Meyer's PR would look like:

```
/table-column/text/
/table-column/text/cell-view
/table-column/text/group-header
/table-column/icon/
/table-column/icon/cell-view
/table-column/icon/group-header
```
Where each folder potentially has index.ts, styles.ts, and template.ts

What about also making the element names align with that heirarchy, i.e.: `table-column-text`, `table-column-text-cell-view` (this PR would need an update), `table-column-text-group-header` (Meyer's PR already this format)?

Reply:
Updated naming as suggested in Milan's last comment. Also updated a few stray references of `cellElement` in variable names to `cellView`.